classes/01class/exercises/c01-git03/gpavelar/README.md

```diff
@@ -0,0 +1,53 @@
+# c01-git03
+
+## Questions (answer inline)
+
+1. Did you notice any difference in the size of the repo before and after adding the big file?
+
+> According to those commands `du -sh` and `du -h` the size of the repo was around 4.0K before `git init` command.
+> After that the repository created locally and run all commands required it was around 170K. The command `git init` creates hidden files under the directory `.git` containing all metadate for this new repository to keep track of everything.
+> Adding a big file into this repository its size increase a bit bit, size around 125M (I created another file a little bit bigger). When you add this file into the repository, .git directory increases its size due to the fact it now trying to store/manage this big file that its size is above git limits. Eventually, GitHub will block pushes that contain files over 100MB.
+
+2. What is the reason for this problem?
+
+> Git was created originally to handle source code that was quite small. Github has a size limit for files to not overload servers and decrease the performance. And it keeps track of any change in a binary file (whole history) as well.
+> So it could be a serious problem when cloning repositories and pushing them.
+> A simple way to overcome it using `git lfs` (git large filesystem) 'extension' on the computer. It creates a pointer to your 'big file' that is stored into git lfs system.
```

Comment:
Great option Gustavo. You are the first to offer this option, feel free to share this info in the slack channel :)

Reply:
Cheers!